src/main/resources/feedbackQuestionRubricSubmissionFormBodyFragment.html

```diff
@@ -1,3 +1,3 @@
 <td class="col-md-1">
-    <input class="overlay" type="radio" ${disabled} id="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}-${col}" name="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}" value="${row}-${col}" ${checked}/><span class="color_neutral overlay"> ${description}</span>
+  <input class="overlay" type="radio" ${disabled} id="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}-${col}" name="${Const.ParamsNames.FEEDBACK_QUESTION_RUBRICCHOICE}-${questionIndex}-${responseIndex}-${row}" value="${row}-${col}" ${checked}/><span class="color_neutral overlay"> ${description}</span>
```

Comment:
There is a `span` tag at the end of this line; it should be moved to the next line.